public/pug/landing.pug

```diff
@@ -4,42 +4,30 @@ block styles
     link(rel="stylesheet" type="text/css" href="../css/landing.css")
 
 block main
-    section(class="container")
-        h1  Be more productive
-        p.
-            "What to do" is an easy to use to-do application, designed for
-            individual or colaborativ use.
+    .container
+        p#logo.landing What to do?
```

Comment:
Don't take the logo out the header. Don't touch the header and add another one to the landing page.

If you want to reuse the styles, put the relevant ones in a class, but don't steal the headers id.

Reply:
By leaving the Logo in the heading, there will be two times the same logo next to itself, which would be redundant. Would't I be just copying and pasting those styles? I wanted to reuse them with this approach

Reply:
> Would't I be just copying and pasting those styles

Put them in a class (that what I meant w/ "reuse the styles").

Reply:
> By leaving the Logo in the heading, there will be two times the same logo next to itself, which would be redundant.

It's about how it looks, not it's information content. And like this it look horrible and super inconsistent, as all other site will have the logo in the header.
The big logo at the beginning of the page looks great, having a small version of it in the corner will not detract from it.

Reply:
An alternative would be removing the header from the landing page. Having a link to the app/login site within the content.